CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -181,8 +184,8 @@ public TableViewPane(final TableViewTopComponent parent) {
         this.table = new TableView<>();
         table.itemsProperty().addListener((v, o, n) -> table.refresh());
         table.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
-        table.setPadding(new Insets(5));
-        setCenter(table);
+        table.setPadding(new Insets(5));      
+        paginate(null);
```

Comment:
Why do you pass null in here initially?

Reply:
The case I happened to be testing before I made the PR was with Table View already open when starting Constellation. Passing null here is more because I couldn't find a better object to pass here (since there doesn't appear to be a useful field I could find containing an array of the data). There probably is a better value to pass.. I just haven't come up with it yet :)

Reply:
Note: If there is a useful field with the data I'm after, there probably won't be a need to pass any parameters into `paginate`